pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -511,6 +511,6 @@ message LogicalReplicationWriterSpec {
     // Checkpoint stores a set of resolved spans denoting completed progress.
     optional jobs.jobspb.StreamIngestionCheckpoint checkpoint = 7 [(gogoproto.nullable) = false];
 
-    // TableDescriptors is a map between from destination table IDs to the source table descriptor.
+    // TableDescriptors is a map from destination table IDs to the source table descriptor.
```

Comment:
While we are fixing this line, the plurality of the the two side are mismatched. Maybe: "from destination table IDs to the source table descriptors"